src/custom/state/application/atom.ts

```diff
@@ -0,0 +1,29 @@
+import { atom } from 'jotai'
+import { atomWithStorage, useUpdateAtom, useAtomValue } from 'jotai/utils'
+
+export { useUpdateAtom, useAtomValue }
+
+type FollowPendingTxPopup = {
+  showPopup: boolean
+  hideFollowTxPopup: boolean
+}
+
+/**
+ * Base atom that store the popup state that indicate how to follow a pending tx
+ */
+export const followPendingTxPopupAtom = atomWithStorage<FollowPendingTxPopup>('followPendingTxPopup', {
+  hideFollowTxPopup: false,
+  showPopup: false,
+})
+
+export const handleFollowPendingTxPopupAtom = atom(null, (_get, set, update: boolean) => {
+  set(followPendingTxPopupAtom, (prev) => ({ ...prev, showPopup: update }))
+})
```

Comment:
Nitpick: Since this is a param and not named, I suggest a small renaming:

```suggestion
export const handleFollowPendingTxPopupAtom = atom(null, (_get, set, showPopup: boolean) => {
  set(followPendingTxPopupAtom, (prev) => ({ ...prev, showPopup }))
})
```

Same applies to the atom below